tensorflow/core/kernels/sequence_ops.cc

```diff
@@ -112,6 +107,14 @@ TF_CALL_int64(REGISTER_GPU_KERNEL);
 
 #endif  // GOOGLE_CUDA
 
+#ifdef TENSORFLOW_USE_SYCL
```

Comment:
Move mos intended to keep CPU, GPU, SYCL order of registration. It's not essential.